tools/aa/commands/FixCommand.java

```diff
@@ -20,15 +20,26 @@
 import com.google.startupos.tools.aa.commands.checks.DummyCheck;
 import com.google.startupos.tools.aa.commands.checks.FailingDummyCheck;
 import com.google.startupos.tools.aa.commands.checks.FixCommandCheck;
+import com.google.startupos.tools.aa.commands.checks.FormattingCheck;
+import java.util.List;
 import javax.inject.Inject;
+import dagger.Lazy;
 
 public class FixCommand implements AaCommand {
 
-  ImmutableList<FixCommandCheck> checks =
-      ImmutableList.of(new DummyCheck(), new FailingDummyCheck());
+  List<FixCommandCheck> checks;
 
   @Inject
-  public FixCommand() {}
+  public FixCommand(
+      Lazy<FormattingCheck> formattingCheck,
```

Comment:
`Lazy` is helpful if you don't always want to instantiate the object, and so you can choose to do so in `.get()`.
Since here you're instantiating always, `Lazy` is not useful, and you can just inject without it.